src/ostorlab/cli/scan/run/run.py

```diff
@@ -121,7 +136,7 @@ def run(
             runtime_instance.scan(
                 title=ctx.obj["title"],
                 agent_group_definition=ctx.obj["agent_group_definition"],
-                assets=None,
+                assets=asset_group if asset_group is None else asset_group.targets,
```

Comment:
I think this is breaking the `--no-asset` flag.

Reply:
how come?, only case would be if you provide the yaml file with the --no-asset. which should be an illegal command to begin with.